sdcm/cluster.py

```diff
@@ -646,15 +657,21 @@ def smp(self):
         --abort-on-internal-error 1 --abort-on-ebadf 1 --enable-sstable-key-validation 1 --smp 6 --log-to-syslog 1
         --log-to-stdout 0 --default-log-level info --network-stack posix"
         """
-        grep_result = self.remoter.run(f'grep "^SCYLLA_ARGS=" {self.scylla_server_sysconfig_path}')
+        try:
+            grep_result = self.remoter.run(f'grep "^SCYLLA_ARGS=" {self.scylla_server_sysconfig_path}')
+        except Exception as exc:
+            self.log.error(f"Failed to get SCYLLA_ARGS. Error: {exc}")
+            return ''
+
         scylla_smp = re.search(r'--smp\s(\d+)', grep_result.stdout)
         self.log.debug(f"SMP on node {self.name}: {scylla_smp.groups() if scylla_smp else scylla_smp}")
 
         return scylla_smp.group(1) if scylla_smp else ''
 
     @property
     def scylla_server_sysconfig_path(self):
-        return f"/etc/{'sysconfig' if self.distro.is_rhel_like else 'default'}/scylla-server"
+        prefix = INSTALL_DIR if self.is_nonroot_install else ''
```

Comment:
let's revert since as I understand we won't run longevities using non-root Scylla installation

Reply:
done